ext/legacydedupefinder/Civi/LegacyFinder/Finder.php

```diff
@@ -3,15 +3,23 @@
 namespace Civi\LegacyFinder;
 
 use Civi\Core\Event\GenericHookEvent;
+use Civi\Core\Service\AutoSubscriber;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 
-class Finder {
+class Finder extends AutoSubscriber implements EventSubscriberInterface {
```

Comment:
FYI AutoSubscriber already implements EventSubscriberInterface

Reply:
@colemanw it finally fired - I've pushed the working version into https://github.com/civicrm/civicrm-core/pull/31689